src/extension/background-script/WelcomeServices/restoreBackup.ts

```diff
@@ -48,6 +50,13 @@ export async function restoreBackup(json: object, whoAmI?: ProfileIdentifier) {
                         )
                     }
                 }
+
+                for (const x of data.wallets) {
+                    const record = WalletRecordFromJSONFormat(x)
+                    if (record.type === 'managed') await importNewWallet(record)
+                    // TODO:
+                    // restore exotic wallets
```

Comment:
No need and no way to "restore exotic wallets"

Reply:
Sure, I'm removing it.